notebooks/experiments/conditional_diffusion/dna_diff_baseline_conditional_UNET.ipynb

```diff
@@ -0,0 +1,7119 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        def update_model_average(self, ma_model, current_model):
</pre><p>We should add <code>nn.Module</code> type hints to <code>ma_model</code> and <code>current_model</code> and <code>None</code> to the method output.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinellolab/DNA-Diffusion/pull/60/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>